symplyphysics/laws/kinematic/accelerated_angular_velocity_from_time.py

```diff
@@ -0,0 +1,74 @@
+from sympy import Eq, dsolve, solve
```

Comment:
Mention that velocity is constant in the file name.
Something like `angular_velocity_from_constant_angular_acceleration`